src/org/ods/component/ScanWithSnykStage.groovy

```diff
@@ -0,0 +1,88 @@
+package org.ods.component
+
+import org.ods.services.SnykService
+
+class ScanWithSnykStage extends Stage {
+  protected String STAGE_NAME = 'Snyk Security Scan'
+  protected SnykService snyk
```

Comment:
is this variable used only internally? If yes `private` would be a better option.